src/main/java/mServer/crawler/BannedFilmFilter.java

```diff
@@ -1,19 +1,74 @@
 package mServer.crawler;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+
 import de.mediathekview.mlib.daten.DatenFilm;
+import de.mediathekview.mlib.tool.Log;
+import mServer.tool.MserverDaten;
+import mServer.tool.MserverKonstanten;
 
 public class BannedFilmFilter {
-
-  public static boolean isBanned(final DatenFilm film) {
-    if (film.arr[DatenFilm.FILM_TITEL].equalsIgnoreCase("Geschichte einer Liebe - Freya")) {
-      return true;
-    }
-    if (film.arr[DatenFilm.FILM_TITEL].equalsIgnoreCase("Wir haben genug - Wirtschaft ohne Wachstum")) {
-      return true;
-    }
-    if (film.arr[DatenFilm.FILM_TITEL].toUpperCase().contains("Auslegung der Wirklichkeit - Georg Stefan Troller".toUpperCase())) {
-      return true;
+  private List<String> bannedTitles = new ArrayList<String>();
+	
+  public BannedFilmFilter() {
+	Log.progress("create BannedFilmFilter from " + MserverDaten.system[MserverKonstanten.SYSTEM_BANNEDFILMLIST_NR] );
+	bannedTitles = new ArrayList<String>();
+	try (BufferedReader reader = new BufferedReader(new InputStreamReader(getInputStreamFromPath(MserverDaten.system[MserverKonstanten.SYSTEM_BANNEDFILMLIST_NR])))) {
```

Comment:
The InputStreamReader will not be closed properly using try-with-resources as it wasnt declared as a „resource“.